@vates/node-vsphere-soap/lib/client.mjs

```diff
@@ -150,8 +164,9 @@ Client.prototype._connect = function () {
         .runCommand('RetrieveServiceContent', { _this: 'ServiceInstance' })
         .once('result', function (result, raw, soapHeader) {
           if (!result.returnval) {
+            const vErr = new VmwareError(err)
```

Comment:
I may be wrong, but isn't it supposed to be `const vErr = new VmwareError(raw)` instead of `VmwareError(err)` ?

Reply:
nice catch , I will check what is inside raw to ensure it can be handled like an error